.github/workflows/daily-wpt-fyi.yml

```diff
@@ -62,6 +62,10 @@ jobs:
           ref: ${{ env.NIGHTLY_REF || steps.setup-node.outputs.node-version }}
       - name: Set env.NODE
         run: echo "NODE=$(which node)" >> $GITHUB_ENV
+      - name: Set env.WPT_DAILY_REF
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: echo "WPT_DAILY_REF=$(gh api /repos/web-platform-tests/wpt/branches/epochs/daily | jq -r '.commit.sha')" >> $GITHUB_ENV
```

Comment:
```suggestion
        run: echo "WPT_DAILY_REF=$(gh api /repos/web-platform-tests/wpt/branches/epochs/daily --jq '.commit.sha')" >> $GITHUB_ENV
```

Reply:
Is it necessary?

Reply:
It's not, it was just a nit. Thanks for taking the suggestion though!